pkg/model/initializers.go

```diff
@@ -393,6 +393,10 @@ func (ml *ModelLoader) grpcModel(backend string, o *Options) func(string, string
 	}
 }
 
+func (ml *ModelLoader) ListAvailableBackends(assetdir string) ([]string, error) {
+	return backendsInAssetDir(assetdir)
```

Comment:
clarification question more than anything else: Is this endpoint is not intended to pick up external GRPC backends?

Reply:
good catch! will update this once at my PC to include listing also external backends